tests/test_misc.py

```diff
@@ -118,8 +118,8 @@ def useless_func() -> int:
 
     def test_diff_environment_yml(self, capsys) -> None:  # type: ignore
         # Test with synthetic environment
-        env = {"dependencies": ["python==3.9", "numpy", "fiona"]}
-        devenv = {"dependencies": ["python==3.9", "numpy", "fiona", "opencv"]}
+        env = {"dependencies": ["python==3.9", "numpy", "pandas"]}
```

Comment:
why did you add pandas instead here?

Reply:
It is just a synthetic list of dependencies for running the tests (so it would still have worked without this change), but I didn't want to leave "fiona" in there to avoid confusion, as it's not a dependency to our packages anymore, so I simply modified the synthetic list using "pandas" instead.